README.md

```diff
@@ -119,3 +119,9 @@ You can either call modules, rulebooks and playbooks by their Fully Qualified Co
 ## License Information
 
 See [LICENSE](./LICENSE) to see the full text.
+
+## Generating the api client
+
+1. Install the [openapi-generator](https://github.com/OpenAPITools/openapi-generator) library
```

Comment:
The last commit introduced a different generator. Is this a new generator? Better?

Reply:
Yes a different generator, this one I would say is more of the "industry standard" and was easier to debug some issues / has much more community support from being so much larger